pages/api/search.ts

```diff
@@ -12,7 +12,10 @@ import siteConfig from '../../config/site.config'
  * @returns Sanitised query string which replaces non-alphanumeric characters with ' '
  */
 function sanitiseQuery(query: string): string {
-  const sanitisedQuery = query.replace(/[^a-zA-Z0-9]/g, ' ')
+  const sanitisedQuery = query
```

Comment:
You are still not removing characters including `'`, which causes 400 errors. Take care of this issue.

Reply:
FYI: https://github.com/spencerwooo/onedrive-vercel-index/issues/302